src/c++/library/http_client.cc

```diff
@@ -1468,7 +1468,12 @@ InferenceServerHttpClient::Infer(
   if (curl_status == CURLE_OPERATION_TIMEDOUT) {
     sync_request->http_code_ = 499;
   } else if (curl_status != CURLE_OK) {
-    sync_request->http_code_ = 400;
+    if (verbose_) {
```

Comment:
Question: For other errors we print to std:err and not use verbose / std:cout - would we want to do the same here for consistency?

Reply:
Done